specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/examples/ClusterExtensionTypes_List.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "aaaaaaaaaa",
+    "resourceGroupName": "rgextensionTypes",
+    "clusterRp": "Microsoft.ContainerService",
+    "clusterName": "aaaaaaaaaaaaaaaaaaaaa",
+    "api-version": "2021-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "releaseTrains": [
+                "aaaaaaaaaaaaaaaaaaaaaa"
+              ],
+              "clusterTypes": "connectedClusters",
+              "supportedScopes": {
+                "defaultScope": "aaaaaaaaaaa",
+                "clusterScopeSettings": {
+                  "properties": {
+                    "allowMultipleInstances": true,
+                    "defaultReleaseNamespace": "aaaaaaaaaaaaaaaaaaaaaaa"
+                  },
+                  "id": "aaaaaaaaaaaaaaaa",
+                  "name": "aaaaaaaaaa",
+                  "type": "aaaaaaaaaaaaaaaaaaa"
+                }
+              }
+            },
+            "systemData": {
+              "createdBy": "aaaaaaaa",
+              "createdByType": "User",
```

Comment:
I assume cluster extensions are always created by SYSTEM and never by USER.

Reply:
We discussed today, this will likely be the "publisher" e.g. Microsoft vs. eventually third party publishers ("User" here is how the tool generated the sample)